testing/types.go

```diff
@@ -1,16 +1,22 @@
 package testing
 
+type SignedArray struct {
+	Signed []uint64
```

Comment:
These names are very confusing given the use of an unsigned type. I don't know what this is intending.

Reply:
I was testing, it should be `int64`.